ceph/utils.py

```diff
@@ -211,8 +211,8 @@ def create_ibmc_ceph_nodes(
                     # can be used only for specific OSD_SCENARIO
                     node_params["osd-scenario"] = node_dict.get("osd-scenario")
 
-                if node_dict.get("image-name"):
-                    node_params["image-name"] = node_dict.get("image-name")
+                if node_dict.get("image-name", {}).get("ibmc"):
```

Comment:
I think it should be `node_dict.get("image-name"`.

Reply:
We want the empty dictionary, so that we get the value in 1 step, as well as not hit KeyNotFoundError

Reply:
Done